SU-SWS/saa_victoria_subtheme
# Adapt 530  vivid red banner links

# READY FOR REVIEW
# Summary
- Change hover colors for regular link and CTA action link in top red banner (Cardinal red background) and vivid red banners (which are now vivid red dark background color)

# Review By (Date)
- 7/30

# Urgency
- medium, accessibility fixes 

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Check top of the page red banner with regular link in a paragraph and CTA link:
- regular link has underline and white text style, then on hover is extra light cloud gray and underlined still
- CTA link is white no underline, then on hover is underlined and extra light cloud gray
4. Check in-page red banner that it:
- is now vivid red DARK color
- regular link has underline and white text style, then on hover is extra light cloud gray and underlined still
- CTA link is white no underline, then on hover is underlined and extra light cloud gray
(sorry figma is not updated yet - I'm hoping Michelle will do that tomorrow. Please let me know if you have a question about how it should look!).
![image](https://user-images.githubusercontent.com/12898896/88859422-f3a2b080-d1b6-11ea-948c-d91ec2fc2133.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-530
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
